environments/dev-env/README.md

```diff
@@ -0,0 +1,394 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+For MacOS only:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
```

Comment:
I suggest that this be worded in a way that makes it clear that this is just one example of how to get these packages installed.

Also, if I understand correctly, the command shown here is an _alternative_ to the command shown just below.  If so, it would help to explicitly say that.

Reply:
You're correct! Just fixed it